src/scenarios/scenarios_4.ts

```diff
@@ -1,16 +1,45 @@
-import {SubstrateService} from "../services/substrate-service";
-import {ScenarioInterface} from "./scenario-interface";
-import Logger from './../services/logger';
+import { SubstrateService } from "../services/substrate-service";
```

Comment:
I've added prettier to the project. Please:
* Add VSCode plugin to use the prettier config from the project
* You can apply changes to the whole project from prettier in scope of this ticket

Reply:
Formated the code.